assets/mods/simplify/mod.js

```diff
@@ -755,19 +776,56 @@
 
 			// add checkboxRightAlign field to options in sc.OPTIONS_DEFINITION
 			const original = sc.OptionRow.prototype.init;
-			sc.OptionRow.prototype.init = function() {
-				original.apply(this, arguments);
-				if (this.option.type === 'CHECKBOX' && this.option.checkboxRightAlign) {
-					this.typeGui.button.hook.align.x = ig.GUI_ALIGN.X_RIGHT;
-					const additionalWidth = this.typeGui.hook.size.x - this.typeGui.button.hook.size.x;
-					const lineHook = this.hook.children[1];
-					const slopeHook = this.hook.children[2];
-					lineHook.size.x += additionalWidth;
-					slopeHook.pos.x += additionalWidth;
+
+			sc.OptionRow.prototype.defaultModIcon = new ig.ImageGui(new ig.Image('media/gui/menu.png'), 536, 160, 23, 23);
+			
+			sc.OptionRow.prototype._addModIcon = function(optionName) {
+				debugger;
+				const iconPath = ig.lang.get('sc.gui.options.' + optionName + '.icon');
+				if (iconPath) {
+					const img = new ig.Image(iconPath);
+					img.addLoadListener({
```

Comment:
A load listener is not necessary. While the `ig.Image` is loading `ig.ImageGui` will render nothing.

Reply:
It is necessary in this case for errors to be caught.